jbs/src/main/java/org/openjdk/skara/jbs/JdkVersion.java

```diff
@@ -77,6 +77,17 @@ private static List<String> splitComponents(String raw) {
             }
         }
 
+        // Check for team/project special version
+        if (finalComponents.isEmpty()) {
+            var projectRepoPattern = Pattern.compile("repo-([a-z0-9]*)");
```

Comment:
For consistency with the surrounding code, would it be better to define this pattern as a static variable?

Reply:
I thought about that and didn't bother, but you are right.